src/encoded/static/components/facets/defaults.js

```diff
@@ -443,19 +443,20 @@ export class DefaultDateSelectorFacet extends React.Component {
         let deleteSubmittedFilter = '';
         let deleteReleasedFilter = '';
         let missingFilter = '';
+        let searchBaseCopy = searchBase;
```

Comment:
I’m getting an ESLint error on line 422 because `searchBase` doesn’t get modified anymore, so it wants a const.

Reply:
Oops, thanks for catching that!